_pages/profile/[id]/submission-details-card/small-avatar.js

```diff
@@ -38,14 +40,47 @@ export default function SmallAvatar({ submissionId }) {
         evidence.file.name.replaceAll(/[^\s\w]/g, "")
           ? evidence.file.name
           : "We are doing some maintenance work and will be online again soon.");
-  return (
+  return submission?.status === "None" && submission?.disputed ? (
```

Comment:
Feels like code is repeated. 
Why not parametrize `<Image variant={aVariant} />`?

Changes would be setting variables like:
`aVariant = (submission?.status === "None" && submission?.disputed) ? 'challengedSmallAvatar' : 'smallAvatar'`

Reply:
Thanks Miguel for the suggestion!
I'll change the code and send a new commit with less redundant code.